blog/welcome-to-developer-development/index.qmd

```diff
@@ -11,3 +11,43 @@ page-layout: full
 The Developer Development
 
 {{< include /_back-blog.qmd >}}
+
+# Week 1
+
+## SE1
+
+#### Summary
```

Comment:
What I wrote for the summary here was..

This first part of the book "Software Engineering at Google" explores the concept of software engineering, distinguishing it from programming and computer science. It highlights Google's unique perspective on software engineering and its role in the
industry. It provides insights into Google's approach to software engineering, emphasizing the importance of adapting and evolving code over time, scaling with growth, and making informed decisions based on experience. It encourages a more rigorous approach to software development and maintenance, reflecting the changing landscape of software in the modern world.

Maybe we could add some of this?